dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -886,7 +887,24 @@ internal class DokkaSymbolVisitor(
 
 
     // ----------- Translators of modifiers ----------------------------------------------------------------------------
-    private fun KtSymbolWithModality.getDokkaModality() = modality.toDokkaModifier()
+    private fun KtSymbolWithModality.getDokkaModality(): KotlinModifier {
+        val isInterface = this is KtClassOrObjectSymbol && classKind == KtClassKind.INTERFACE
+        return if (isInterface) {
+            when (modality) {
+                // modifiers other than "sealed" are redundant for interfaces
```

Comment:
Does this check make sense? Which user cases does it covers?

Reply:
For just `interface` compiler will return modality `abstract`